### ISSUE TITLE
Some cleanup

### ISSUE THREAD

**Issue by mauran** [emilstahl/ipv6#7]
I really think this small codebase could become a lot cleaner. Any suggestions @lasserafn?

**Comment by LasseRafn** [comment]
True, I’d gladly cleanup some stuff 👍  personally I would throw the data reading into a separate file/class initially
